### ISSUE TITLE
Unable to enable GPU training

### ISSUE THREAD

**Issue by 3050226203** [mindverse/Second-Me#251]
Here is the training log:
***** Running training *****
Num examples = 7
Num Epochs = 3
Instantaneous batch size per device = 2
Total train batch size (w. parallel, distributed & accumulation) = 4
Gradient Accumulation steps = 2
Total optimization steps = 6
Number of trainable parameters = 4,399,104
0%| | 0/6 [00:00<?, ?it/s]2025-04-16 12:42:10 [INFO] train.py:276 - === Training Begin ===
2025-04-16 12:42:10 [INFO] train.py:276 - === Training Begin ===
2025-04-16 12:42:10 [INFO] train.py:277 - Checking initial conditions:
2025-04-16 12:42:10 [INFO] train.py:277 - Checking initial conditions:
2025-04-16 12:42:10 [INFO] train.py:298 -
=== Starting Step 1 ===
2025-04-16 12:42:10 [INFO] train.py:298 -
=== Starting Step 1 ===
2025-04-16 12:42:10 [INFO] train.py:304 - CPU Usage: 0.0%
2025-04-16 12:42:10 [INFO] train.py:304 - CPU Usage: 0.0%
2025-04-16 12:42:10 [INFO] train.py:305 - Memory Usage: 763.52MB
2025-04-16 12:42:10 [INFO] train.py:305 - Memory Usage: 763.52MB
2025-04-16 12:42:10 [INFO] train.py:308 - Thread Count: 18
2025-04-16 12:42:10 [INFO] train.py:308 - Thread Count: 18
2025-04-16 12:42:10 [INFO] trainprocess_service.py:899 - Training started
/usr/local/lib/python3.12/site-packages/transformers/trainer.py:3618: FutureWarning: `torch.cpu.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cpu', args...)` instead.
ctx_manager = torch.cpu.amp.autocast(cache_enabled=cache_enabled, dtype=self.amp_dtype)
2025-04-16 12:42:10 [INFO] trainprocess_service.py:945 - Progress updated: 0% - Current step: 0/6

The project is in docker, and the NVIDIA Container Toolkit has been installed.The project could not start the GPU

**Comment by yingapple** [comment]
Our latest version doesn't support GPU.
However, the open source community opened a PR for this problem. https://github.com/mindverse/Second-Me/pull/228
